### DISCUSSION TITLE
How much more fiddling and hacking can you guys take?

### DISCUSSION THREAD

**Discussion by 0x00A1FA** [vuejs/language-tools#3261] [Polls]
Hello,

I was wondering if I'm the only one that's getting annoyed by the constant fiddling, updates, fixes and other stuff that's constantly causing code refactorings and unnecessary code updates.

For example, we recently got `defineModel` and `defineProps` working together (https://github.com/vuejs/language-tools/issues/3164). This would have been a great start for Vue 3.3, but we only get it now. Another one would be generic props, as the beforementioned issue explores.

I feel like I'm missing out on the amazing React ecosystem and DX (no wonder since TSX is maintained by Microsoft).

Check out the poll below and share your thoughts.

Thanks!

**Comment by laterdayi** [comment]
I'm not sure why this happened, but I really appreciate the efforts of the authors and contributors, and I look forward to stabilizing support for vue3.3.* as soon as possible, now upgraded to 3.*, too many types of errors

**Comment by johnsoncodehk** [comment]
This is almost my one-person project, yes I'm not professional, I voted it up, hope you are satisfied. :)
